service/models/operations/optimize.py

```diff
@@ -59,13 +59,26 @@ def gen_risk_bound(self):
             return -self.risk_bound
 
 
-def objfun(x, initial_guess, objective_function_option):
-    if objective_function_option == "lower_bound":
-        return np.sum(np.abs(x))
-    if objective_function_option == "upper_bound":
-        return -np.sum(np.abs(x))
-    if objective_function_option == "initial_guess":
-        return np.sum(np.abs(x - initial_guess))
+def objfun(x, initial_guess, objective_function_option, relative_importance):
```

Comment:
As this is getting more complicated it may be worthwhile adding a comment above this explaining what it is youre looking for.

Reply:
will add!